### DISCUSSION TITLE
Savage Worlds Adventure Edition by @FilBot3

### DISCUSSION THREAD

**Discussion by sigrunixia** [Obsidian-TTRPG-Community/fantasy-statblocks#168] [Show and tell]
This thread is for @FilBot3 and their Savage Worlds Adventure Edition Statblock

---

# Savage Worlds Adventure Edition

I've attempted to create a Custom Layout for the SWADE system for both a PC (implemented) and for Extras (not implemented). Inside the `.zip` file is a README, CHANGELOG, the JSON template, and the example image. The README will contain the YAML schema for the `statblock`.

![SWADE_PC_Example](https://user-images.githubusercontent.com/3039509/197441871-a38c8e56-9fc9-4e98-83a7-ea10de1c7ae1.png)

[TTRPG_Statblock_Layout_SWADE.v1.1.1.zip](https://github.com/valentine195/obsidian-5e-statblocks/files/9848568/TTRPG_Statblock_Layout_SWADE.v1.1.1.zip)

### October 2022 Updates
https://github.com/valentine195/fantasy-statblocks/discussions/41#discussioncomment-3946653 - I'll be incrementing the version when I get the Extra Layout completed as well.
https://github.com/valentine195/fantasy-statblocks/discussions/41#discussioncomment-3946667 - Current issues: traits that are in an Inline Group do not seem to render properly. I'm sure it's something with how I have the blocks nested. No CSS to color things properly.
https://github.com/valentine195/fantasy-statblocks/discussions/41#discussioncomment-3946938 - So, the traits block doesn't seem it works in an Inline Group. This may be a bug/issue. So, for dynamically generated sections that use trait these need to be their own Group that is not nested in an Inline Group.
https://github.com/valentine195/fantasy-statblocks/discussions/41#discussioncomment-3947004 - Bumped to v1.1.0 because all elements are showing, just not in the columns I want them in. However, this gets a rough sheet working that could be used for both PC's and Extras.

---

**Original Thread**: https://github.com/valentine195/fantasy-statblocks/discussions/41#discussioncomment-3946647

**Comment by General-Anubis** [comment]
I'm really new to all this, so I'm not sure how to do it myself - any chance to get the DiceRoller built into this for the stats and such?

**Reply by sigrunixia** [reply]
This is originally a layout by @Filbot3, but I will see what I can do this weekend to "turn on" dice rolls. 

Caveat: No matter what, you'll still need to fill out the code block appropriately.

**Reply by sigrunixia** [reply]
@General-Anubis Here is an updated layout that enables the parse for dice. You will still need to use specific dice terminology as defined in [our documentation](https://plugins.javalent.com/statblocks/readme/code-block#Dice+Rolls).

[SWADE PCs.json.zip](https://github.com/javalent/fantasy-statblocks/files/11585188/SWADE.PCs.json.zip)


Edit: You may also need to use callbacks for some of the rolls if you need a different roll type.

**Reply by General-Anubis** [reply]
Greatly appreciate it!

**Reply by FilBot3** [reply]
Yeah, sorry. I didn't know how to make the dice rolls work and it worked fine for my process at the time. Thank you for adding the dice roller capability, @sigrunixia

**Reply by sigrunixia** [reply]
> Yeah, sorry. I didn't know how to make the dice rolls work and it worked fine for my process at the time. Thank you for adding the dice roller capability, @sigrunixia

No problem, and _A LOT_ has changed in Statblocks since the layout was originally made, so do not feel bad.
